nuget/dotnet/rl.net.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <title>RL.Net</title>
```

Comment:
since the assembly name changed to `rl.net` do we want this to change too?

Reply:
The namespace of the package itself is still Rl.Net so I think it is more consistent to keep it as is here. The assembly name itself was changed to lower case as not all filesystems are case sensitive and now that this is cross platform that was proving troublesome.